### ISSUE TITLE
Some questions while running the code

### ISSUE THREAD

**Issue by DangBowen-Bell** [zkf1997/COINS#3]
Hi, thank you for your open source of this great work.

There are some questions:

1. When I run scene.py to visualize the scene, I get an error from the following line saying that there is no file of 'X_semantic.ply' (X refers to the scene name). 
 
> https://github.com/zkf1997/COINS/blob/5663a53471dce8e536363cfe09bdabd447056974/data/scene.py#L198

I am sure I have downloaded all the required data. I am wondering if there are some additional data not specified in the README of this repo.

2. To run the interaction synthesis using pretrained model, I replace the semantic scene with original scene to skip the error above. However, as we can see, the result is very bad.

![image](https://user-images.githubusercontent.com/35923383/188876806-7054ab25-b379-4c98-b880-b6243cd42c99.png)

I do not think it is the problem of the scene.

I am looking for your reply.

**Comment by zkf1997** [comment]
Thanks for your questions! Please find my responses below:

Question 1:
This is a problem caused by different file names in the older vesion and the current version of the PROX-E dataset. I made a push to completely use the new suffix '_withlabels' instead of the old '_semantic' in PROX-E.  Please delete the scene cache files and build the scenes again. You can check the scene segmentation results in the rendered `scene_segmentation/*_semantic.png`. Thanks for pointing this out!


Question 2:
May you run it again after confirming the scene segmentation is succeded? The interaction synthesis does not work without correct segmentation. If you still have this problem, please send me the command and also the log so that I can better understand what happens.

**Comment by DangBowen-Bell** [comment]
Thank you for your quick response. 

I use the new code to generate the scene data again and the result is still very bad.

My command and log are as follows:

Command:

`python two_stage_sample.py --exp_name test --lr_posa 0.01 --max_step_body 100  --weight_penetration 20 --weight_pose 10 --weight_init 0.01  --weight_contact_semantic 1 --num_sample 8 --num_try 1  --visualize 1 --full_scene 1 --interaction 'sit on-chair+touch-table' --scene_name 'MPH16' --composition 1 `

Log:

loading from cache for: MPH1Library
loading from cache for: MPH8
loading from cache for: N0SittingBooth
loading from cache for: N0Sofa
loading from cache for: N3Library
loading from cache for: N3Office
loading from cache for: N3OpenArea
loading from cache for: Werkraum
GeForce RTX 3080
/home/ylabcg/anaconda3/envs/dbw_coins/lib/python3.7/site-packages/scipy/sparse/_index.py:82: SparseEfficiencyWarning: Changing the sparsity structure of a csr_matrix is expensive. lil_matrix is more efficient.
  self._set_intXint(row, col, x.flat[0])
body representation of mesh with  655  points
  0%|                                                                            | 0/1 [00:00<?, ?it/ssit on-chair-3+touch-table-4                                                      | 0/1 [00:00<?, ?it/s]
../interaction/interaction_model.py:279: UserWarning: To copy construct from a tensor, it is recommended to use sourceTensor.clone().detach() or sourceTensor.clone().detach().requires_grad_(True), rather than torch.tensor(sourceTensor).
  torch.tensor(self.args.template_body, device=obj_points.device).unsqueeze(0).expand(B, -1, -1)
/home/ylabcg/anaconda3/envs/dbw_coins/lib/python3.7/site-packages/smplx/body_models.py:280: UserWarning: To copy construct from a tensor, it is recommended to use sourceTensor.clone().detach() or sourceTensor.clone().detach().requires_grad_(True), rather than torch.tensor(sourceTensor).
  param[:] = torch.tensor(params_dict[param_name])
/data/dbw/code/COINS/results/two_stage/test/optim_learned_by_part/sit on-chair+touch-table/MPH16/sit on-chair-3+touch-table-4.pkl

It seems that there is no error while running?

**Comment by zkf1997** [comment]
May you first try running `python two_stage_sample.py --exp_name test --lr_posa 0.01 --max_step_body 100  --weight_penetration 100 --weight_pose 10 --weight_init 0.01  --weight_contact_semantic 1 --num_sample 8 --num_try 1  --visualize 1 --full_scene 1 --interaction 'sit on-chair' --scene_name 'MPH16'` and see the results? I would like to debug step by step.

**Comment by zkf1997** [comment]
looks like some pytorch related problem, can you put your pytorch version? I will take a further look

**Comment by DangBowen-Bell** [comment]
My pytorch version is 1.12.1.

**Comment by zkf1997** [comment]
I tried pytorch==1.12.1 and reproduced the error. It seems to be some pytorch problem and I see some other posts about this error. You may try the latest nightly build of pytorch to see whether it fixes the problem or use the tested pytorch==1.11.0. 
I modified the environment.yml to use the specified version of pytorch. In addition, I updated the instructions of synthesis for compositional synthesis, it should use model checkpoints trained only on atomic data that you can download now from the checkpoint sharing link.  
May you try the updated environment and synthesis commands?

**Comment by DangBowen-Bell** [comment]
I set up the environment using the new file and the result is fine now.

Atomic interaction synthesis:
![image](https://user-images.githubusercontent.com/35923383/189257417-815443ea-e7ae-4e3c-99b7-160b34477241.png)

Compositional interaction synthesis: 
![image](https://user-images.githubusercontent.com/35923383/189258195-083dade9-a6f7-42f9-8af2-43e4978a535b.png)

Thank you so much for your patient instruction!
